gbpservice/neutron/tests/unit/nfp/configurator/agents/test_generic_config.py

```diff
@@ -356,9 +339,5 @@ def test_handle_configure_healthmonitor_genericconfigeventhandler(self):
         """
 
         ev = fo.FakeEventGenericConfig()
-        ev.id = 'CONFIGURE_HEALTHMONITOR'
+        ev.id = const.EVENT_CONFIGURE_HEALTHMONITOR
         self._test_handle_periodic_event(ev)
-
-
```

Comment:
This is unnecessary for run_tests script. We do not use unittest but basetest from neutron tests.